databases/week1/w1.sql

```diff
@@ -0,0 +1,17 @@
+SELECT COUNT(task.id) FROM task;
+SELECT * FROM task;
+SELECT COUNT(task.id) FROM task WHERE due_date IS NULL ;
+SELECT * FROM task JOIN status ON task.status_id = status.id WHERE status.name like "done";
+SELECT * FROM task JOIN status ON task.status_id = status.id WHERE status.name not like "done";
+--5--Get all the tasks, sorted with the most recently created first
+SELECT * FROM task ORDER BY created DESC ;
+--6--Get the single most recently created task
+SELECT * FROM task ORDER BY created DESC LIMIT 1;
+--7--Get the title and due date of all tasks where the title or description contains database
+SELECT title, due_date FROM task where CONCAT(title, description) like "%database%" ;
```

Comment:
The `CONCAT` method creates a single value from the values that are passed, so in this case what would happen is that the title, lets say `Title1` and the description, `Description1`, will be united like this `Title1Description1`.

The solution here should include a `OR` statement, e.g. `WHERE title LIKE '%database%' OR description LIKE '%database%'`

Reply:
So thanks for great comments